### ISSUE TITLE
Permission Denial: reading com.google.android.apps.photos.contentprovider.impl.MediaContentProvider

### ISSUE THREAD

**Issue by kyleabens** [capawesome-team/capacitor-file-picker#84]
All works well on iOS but I get the following error when trying to display the selected video on Android in a video tag in my html.  

`java.lang.SecurityException: Permission Denial: reading com.google.android.apps.photos.contentprovider.impl.MediaContentProvider uri content://com.google.android.apps.photos.contentprovider/0/2/content%3A%2F%2Fmedia%2Fexternal%2Fvideo%2Fmedia%2F2523/ORIGINAL/NONE/video%2Fmp4/394201964#t=0.1 from pid=7125, uid=10270 requires the provider be exported, or grantUriPermission()`

I have tried searching for an answer for a couple days now and nothing has worked.  Wondering if you can provide some insight into this error and if any others have had this issue in the past.

```
const result = await FilePicker.pickMedia({
  multiple: true
});
result.files.forEach(file => {
  const url = Capacitor.convertFileSrc(file.path);
  // url is used as src in html
});
```

**Comment by KANekT** [comment]
use https://capacitorjs.com/docs/apis/filesystem#requestpermissions

**Comment by robingenz** [comment]
@kyleabens I'm not sure if it's the same bug but unfortunately embedding locale videos in the img tag on Android doesn't work at the moment, see https://github.com/ionic-team/capacitor/issues/6021

@KANekT Can you share your use case where you need to request permission? In my tests on Android (Android 13, Pixel 5) and iOS, request permissions has not been necessary so far. If you really need this, then i have to implement the method in this plugin.

**Comment by KANekT** [comment]
sorry, I did not know that such a bug already exists in the core  Capacitor

I thought it was just a file permissions issue.

**Comment by kyleabens** [comment]
@KANekT I tried adding requestPermission and that unfortunately didn't resolve my issue.

@robingenz What I'm doing is giving users the ability to select images and videos and then display them in a modal that they can then upload to our server.  Images work fine on Android, it's only videos that are not displaying.  Also videos are not being placed in an img tag, I'm using the video tag for those (Eg. `<video><source src="path"></video>`).  I don't want to convert the videos to base64 since that's memory intensive and unnecessary for simply displaying them before uploading.

Again this works on browser and iOS no problem.  I'm assuming it has something to do with the videos I'm selecting being on Google photos?  However images work fine so idk.  At the end of the day I'm just trying to display a video in the html based on the path of that video

**Comment by kyleabens** [comment]
Might have to use something like this for the time being [https://github.com/jepiqueau/capacitor-video-player](https://github.com/jepiqueau/capacitor-video-player)

**Comment by robingenz** [comment]
> What I'm doing is giving users the ability to select images and videos and then display them in a modal that they can then upload to our server. Images work fine on Android, it's only videos that are not displaying. Also videos are not being placed in an img tag, I'm using the video tag for those (Eg. <video><source src="path"></video>).

Yes, that's the issue i linked: https://github.com/ionic-team/capacitor/issues/6021

One of my clients has the same problem. Therefore, this plugin will probably generate thumbnails for videos in the next version, which can be displayed to the user. Maybe you have another use case and this isn't a solution for you.
